sycl/test/abi/preview_lib_marker_windows.cpp

```diff
@@ -0,0 +1,28 @@
+// RUN: not %clangxx -fsycl %s -o %t 2>&1 | FileCheck --check-prefix=%{CHECK-NO-PREVIEW} %s
```

Comment:
```suggestion
// RUN: not %clangxx -fsycl %s -o %t 2>&1 | FileCheck --check-prefix=CHECK-NO-PREVIEW %s
```

Not sure where `%{...}` came from.  Possibly my bad.

Reply:
This was an oopsie from my side after trying to make it one file. It has been amended.